compiler/qsc_linter/src/lints/hir.rs

```diff
@@ -0,0 +1,15 @@
+use crate::linter::hir::declare_hir_lints;
+
+use super::push_lint;
+
+declare_hir_lints! {
+    (Stump, LintLevel::Allow, "remove this stump after addding the first HIR lint"),
```

Comment:
I know this is just a placeholder, but to confirm HIR lints show up it would be good to include in one of your unit tests.

Reply:
Just added a unit test for the placeholder, and there was indeed a bug. Nice catch!